code/game/objects/items/crayons.dm

```diff
@@ -380,6 +380,9 @@
 	if(edible && (M == user))
 		user << "You take a bite of the [src.name]. Delicious!"
 		var/eaten = use_charges(5)
+		if(!reagents.total_volume) //Prevents divsion by zero
```

Comment:
now that I think about it, why not just

    if(check_empty(user))
        return

it feels like it would do the same thing?